src/Tools/ExternalAccess/Razor/EmptyTextLoader.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Text;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.Razor
+{
+    internal class EmptyTextLoader : TextLoader
+    {
+        private readonly string _filePath;
+        private readonly VersionStamp _version;
+
+        public EmptyTextLoader(string filePath)
+        {
+            _filePath = filePath;
+            _version = VersionStamp.Create(); // Version will never change so this can be reused.
```

Comment:
>Create() [](start = 36, length = 8)

Isn't `VersionStamp.Default` more appropriate?

Reply:
Could be, just brought over from Razor land 😄